releasenotes/notes/1.3/paulifeaturemap-takes-dictionary-as-entanglement-02037cb2d46e1c41.yaml

```diff
@@ -1,9 +1,9 @@
 ---
 fixes:
   - |
-    Fixed that the entanglement in :class:`.PauliFeatureMap` and :class:`.ZZFeatureMap`
+    Previously, the entanglement in :class:`.PauliFeatureMap` and :class:`.ZZFeatureMap`
     could be given as ``List[int]`` or ``List[List[int]]``, which was incompatible with the fact
-    that entanglement blocks of different sizes are used. Instead, the entanglement can be 
+    that entanglement blocks of different sizes are used. Now, the entanglement can be 
     given as dictionary with ``{block_size: entanglement}`` pairs.
```

Comment:
I'm okay with deleting the fix section, since it already mentions the specification of entanglement with a dictionary down below. I will remove it if @Cryoris agrees.

Reply:
Let's delete this -- 1.3 doesn't "fix" the old path of giving the entanglement as `List[List[int]]`, the old path is just invalid and will still fail.